src/routes/relay/entities/schema/sponsored-call.schema.spec.ts

```diff
@@ -75,6 +77,34 @@ describe('sponsoredCall schema', () => {
     });
   });
 
+  it('should validate and predict the Safe address from a valid creation call', async () => {
+    const chainId = '5';
+
+    const to = getProxyFactoryDeployment({
+      network: chainId,
+    })?.defaultAddress;
+
+    const data = {
+      chainId,
+      to,
+      data: MOCK_CREATE_PROXY_WITH_NONCE_TX_CALL_DATA,
+      gasLimit: faker.random.numeric(),
+    };
+
+    const result = await SponsoredCallSchema.safeParseAsync(data);
+
+    expect(result.success).toBe(true);
+
+    if (!result.success) {
+      return;
+    }
```

Comment:
This should never happen right? We have the assertion above that should throw.

Reply:
No, it's just to appeases TypeScript. I've added comments explaining this.